Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/ModalInfoView/ModalInfoViewController.swift

```diff
@@ -50,7 +50,8 @@ public final class ModalInfoViewController: UIViewController {
     
     @IBOutlet private var bodyLabel: UILabel! {
         didSet {
-            bodyLabel.text = viewModel.body.value
+            bodyLabel.attributedText = viewModel.body
+            bodyLabel.textColor = .gdsGrey
```

Comment:
Please can we get a unit test for this change?

Reply:
Added a test for the `textColor`
When looking at the help window for `attributedText` it says the following:
![image 2023-10-13 at 16 24 08](https://github.com/alphagov/di-mobile-ios-common/assets/90717175/ace63556-487b-4614-839d-04a6be32f5fd)
that formatting is removed, not sure if we want to add a test for the text formatting as well?